src/fe-text/gui-readline.c

```diff
@@ -638,7 +653,8 @@ static gboolean paste_timeout(gpointer data)
 {
 	paste_was_bracketed_mode = paste_bracketed_mode;
 
-	if (paste_line_count == 0) {
+    /* Take into account the fact that a line may be split every LINE_SPLIT_LIMIT characters */
+	if (paste_line_count == 0 && paste_buffer->len / LINE_SPLIT_LIMIT <= paste_verify_line_count) {
```

Comment:
What would that flag do? I think sharing the verify length count is fine here, they become the same in the end anyway?